core-web/libs/block-editor/src/lib/extensions/asset-form/asset-form.extension.ts

```diff
@@ -70,6 +71,10 @@ export const BubbleAssetFormExtension = (viewContainerRef: ViewContainerRef) =>
     }
 
     function onHide(editor): void {
+        if (preventClose) {
+            return;
```

Comment:
We need to tell the user here that they can close this because the video is getting uploaded.

Also in the loading indicator below, we need to add a message: "Don't close this window while the video uploads"

What we can do if that the user try to close it... we animate that message, to tell the user "READ THIS"

Reply:
I added it.